invenio_queues/cli.py

```diff
@@ -55,10 +54,11 @@ def declare(queues):
 
 @queues.command('purge')
 @click.argument('queues', nargs=-1)
+@click.option('--force', is_flag=True, default=False)
 @with_appcontext
-def purge_queues(queues=None):
+def purge_queues(force, queues=None):
```

Comment:
```suggestion
def purge_queues(force=False, queues=None):
```

I think for keeping things safe, I would add all around the different function signatures `False` as the default